crates/circuit/src/imports.rs

```diff
@@ -124,18 +118,8 @@ pub fn populate_std_gate_map(py: Python, rs_gate: StandardGate, py_gate: PyObjec
 
 #[inline]
 pub fn get_std_gate_class(py: Python, rs_gate: StandardGate) -> PyResult<PyObject> {
-    let gate_map = unsafe {
-        STDGATE_PYTHON_GATES.get_or_init(py, || {
-            // A fixed size array is initialized like this because using the `[T; 5]` syntax
-            // requires T to be `Copy`. But `PyObject` isn't Copy so therefore Option<PyObject>
-            // as T isn't Copy. To avoid that we just list out None STANDARD_GATE_SIZE times
-            let array: [Option<PyObject>; STANDARD_GATE_SIZE] = [
-                None, None, None, None, None, None, None, None, None, None, None, None, None, None,
-                None, None, None, None,
-            ];
-            array
-        })
-    };
+    let array: [Option<PyObject>; STANDARD_GATE_SIZE] = core::array::from_fn(|_| None);
+    let gate_map = unsafe { STDGATE_PYTHON_GATES.get_or_init(py, || array) };
```

Comment:
Shouldn't we initialize array inside the closure as part of `get_or_init`? Otherwise we'll be creating the array of `None` every time this function is called even though we don't need it.

Reply:
Oh yeah it is called frequently. I thought about this but somehow ignored it.